website/search.py

```diff
@@ -53,14 +53,17 @@ def handle_keypress(self, e: events.KeyEventArguments) -> None:
     def handle_input(self, e: events.ValueChangeEventArguments) -> None:
         async def handle_input():
             with self.results:
-                results = await ui.run_javascript(f'return window.fuse.search("{e.value}").slice(0, 50)', timeout=6)
+                results = await ui.run_javascript(f'return window.fuse.search("{e.value}").slice(0, 100)', timeout=6)
                 self.results.clear()
                 for result in results:
-                    href: str = result['item']['url']
-                    with ui.element('q-item').props('clickable') \
-                            .on('click', lambda href=href: self.open_url(href), []):
-                        with ui.element('q-item-section'):
-                            ui.label(result['item']['title'])
+                    if result['item']['content']:
+                        href: str = result['item']['url']
+                        with ui.element('q-item').props('clickable') \
+                                .on('click', lambda href=href: self.open_url(href), []):
+                            with ui.element('q-item-section'):
+                                ui.label(result['item']['title']).style('font-weight: 500')
+                                ui.label(result['item']['content'][:200] + '...').classes('text-grey')
```

Comment:
It would be great if we could render the docstrings formatted. But `ui.markdown` does not good: docstrings are reStructuredText :-(
Any ideas?

Reply:
I think I might be able to convert it to markdown on the fly but the issue is that we might need to add an extra script for that, could be useful later tho but I would need some ideas on where I should put this `rst2md` translator code.

Reply:
I found that using `ui.markdown` still shows better descriptions than using just a `ui.label`, some things that are common between both rst and markdown do get shown properly and look better than before. I will be doing a commit with that change, I will also change the color for the descriptions back to `white` instead of `grey` since the grey color might be hard to read for some people under some light conditions.

Reply:
@rodja I fixed the results not opening in a new tab, we can now open them with in a new tab with the middle mouse button or with the "Open in a new tab" option from the right click context menu. Unfortunately, the separator between results seems to be gone.